remote-method/USAGE

```diff
@@ -0,0 +1,6 @@
+Description:
+  Creates a new remote method in the LoopBack application.
+
+Example:
+
+  slc loopback:remote-method
```

Comment:
@superkhau what are your concerns regarding `slc loopback:method`? What other kinds of methods with a generator do you envision?

I'd rather avoid maintaining aliases.